pkgs/development/python-modules/botocore/default.nix

```diff
@@ -3,34 +3,37 @@
   awscrt,
   buildPythonPackage,
   fetchPypi,
+
+  # build-system
+  setuptools,
+
+  # dependencies
   jmespath,
-  jsonschema,
-  pytestCheckHook,
   python-dateutil,
-  pythonOlder,
-  setuptools,
   urllib3,
+
+  # tests
+  jsonschema,
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "botocore";
-  version = "1.35.29"; # N.B: if you change this, change boto3 and awscli to a matching version
+  version = "1.35.30"; # N.B: if you change this, change boto3 and awscli to a matching version
   pyproject = true;
 
-  disabled = pythonOlder "3.8";
-
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-TtKKsDZ1uwCKKQxFLF3deqpdTj+hkSqtvfkwV+6ENis=";
+    hash = "sha256-q1NQ6KUOSNNx+i1RfWXCmkDEN4jLmhU4f5PqxaI98P0=";
   };
 
-  pythonRelaxDeps = [ "urllib3" ];
+  # pythonRelaxDeps = [ "urllib3" ];
```

Comment:
What's up with this line?

Reply:
It works without it. I forgot to remove it.
Now it's done.